src/docs/sdk/sample-format.mdx

```diff
@@ -0,0 +1,341 @@
+
+If for some reasons, the profiler implementation doesn't allow that, set it to
+the closest time to the profiler start (or to whatever the profiler gives you as
+a profile start timestamp) and it will be used to align with the transaction
+later on.
+
+`transaction`
+
+: *Object, required.* Contains information about the transaction it is
+associated with.
+
+It can have the following attributes:
+- `id`: *String, required.* Contains the transaction ID.
+- `name`: *String, required.* Contains the name of the transaction.
+- `trace_id`: *String, required.* Contains the trace ID the transaction is part
+  of.
+- `active_thread_id`: *String, required.* Contains the thread ID the transaction
+  was started on as `uint64` converted to a string. On mobile, this is usually
+  the main thread.
+
+`version`
+
+: *String, required.* Represents the version of the Sample format.
+
+Acceptable values are:
+- `"1"`
+
+### Profile data
+
+The profile data consists of a list of samples referring to stacks which in
+terms refers to frames as well as some metadata about threads. This is to keep the amount of data we transfer to the
+server as limited as possible.
+
+```json
+{
+  "profile": {
+    "samples": [
+      { ... },
+      ...
+    ],
+    "stacks": [
+      [ ... ],
+      ...
+    ],
+    "frames": [
+      { ... },
+      ...
+    ],
+    "thread_metadata": {
+      "259": {
+        ...
+      }
+    }
+  }
+}
+```
+
+When collecting a stack trace, you will:
+1. Capture the stack trace.
+2. Capture a timestamp relative to the start of the profile (set in the
+`timestamp` field).
+3. Capture the thread ID where this stack trace was captured.
+4. Create a list of indices for each frames (add it to the frame list if needed
+or use the existing frame's index).
+5. Check if that stack exists in the list of stacks and if it doesn't, add it to the list of stacks
+6. Create a sample containing the thread ID, the relative timestamp and the
+stack ID.
+
+`samples`
+
+: *List, required* Contains a list of sample object captured during execution.
+Each sample can contain the following attributes:
+
+- `elapsed_since_start_ns`: *String, required.* Contains the relative timestamp in nanoseconds since the value contained
+in the `timestamp` field when the sample was captured.
+- `stack_id`: *Integer, required.* Contains the stack index from the `stacks` list.
+- `thread_id`: *String, required.* Contains the thread ID on which the sample was captured.
+
+```json
+{
+  "elapsed_since_start_ns": 1234567890,
+  "stack_id": 1,
+  "thread_id": "259",
+}
+```
+
+`stacks`
+
+: *List, required*. Contains a list of frame indices forming a stack trace.
+
+Stacks should be ordered from outermost to innermost frame.
+
+`frames`
+
+: *List, required*. Contains a list of frame objects (see [Frame
+Attributes](/sdk/event-payloads/stacktrace/#frame-attributes)).
+
+Technically, all attributes are optional so we won't reject the payload if some
+are missing.
```

Comment:
Is that true, though? If there is absolutely no frame information, what is shown then?

Reply:
Maybe just say the rules are the same as for standard event stack frames & symbolication?